tests/admin-ui-tests/utils.ts

```diff
@@ -21,6 +22,26 @@ const promiseSignal = (): Promise<void> & { resolve: () => void } => {
 };
 const projectRoot = findRootSync(process.cwd());
 
+export const makeGqlRequest = async (query: string, variables?: Record<string, any>) => {
+  try {
+    const { errors } = await fetch('http://localhost:3000/api/graphql', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        query,
+        variables,
+      }),
+    }).then(res => res.json());
+    if (errors) {
+      throw errors;
```

Comment:
You might want to make an actual `Error` here with the message being something like ``graphql blah: ${graphql errors.map(x=>x.message).join('\n')}`` so that you get a stacktrace